### PR TITLE
Patch 0.19.2

### PR DESCRIPTION
**Pull request by Pavan-Gunda**
**What this PR does / why we need it**:
This PR adds new patches to existing release
**Which issue this PR fixes** *(use the format `fixes #<issue number>(, fixes #<issue_number>, ...)` to automatically close the issue when PR gets merged)*: fixes https://github.com/elastisys/ck8s-ops/issues/1522

**Public facing documentation PR** *(if applicable)*

**Special notes for reviewer**:

**Add a screenshot or an example to illustrate the proposed solution:**

**Checklist:**

- [ ] Added relevant notes to [WIP-CHANGELOG.md](https://github.com/elastisys/compliantkubernetes-apps/blob/main/WIP-CHANGELOG.md)
- [ ] Proper commit message prefix on all commits
- [ ] Updated the [public facing documentation](https://github.com/elastisys/compliantkubernetes)
- Is this changeset backwards compatible for existing clusters? Applying:
    - [x] is completely transparent, will not impact the workload in any way.
    - [ ] requires running a migration script.
    - [ ] will create noticeable cluster degradation.
          E.g. logs or metrics are not being collected or Kubernetes API server
          will not be responding while upgrading.
    - [ ] requires draining and/or replacing nodes.
    - [ ] will change any APIs.
          E.g. removes or changes any CK8S config options or Kubernetes APIs.
    - [ ] will break the cluster.
          I.e. full cluster migration is required.
- Chart checklist (pick exactly one):
    - [x] I upgraded no Chart.
    - [ ] I upgraded a Chart and determined that no migration steps are needed.
    - [ ] I upgraded a Chart and added [migration steps](https://github.com/elastisys/compliantkubernetes-apps/blob/main/migration).

**Pipeline config** *(if applicable)*
If you change some config options (e.g. add/rename variable or change the default value) you may need to update the config used by the pipeline in `pipeline/config`.

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -18,6 +19,18 @@
 - [v0.5.0](#v050---2020-08-06)
 <!-- END TOC -->
 
+-------------------------------------------------
+## v0.19.2 - 2022-03-17
+
+
+### Changed
+- Added the repo - "quay.io/jetstack/cert-manager-acmesolver" in allowrepo safeguard by default.
+- Backup operator namespaces can for example be added as veloro parameters to be able to back them up. 'alertmanager' is added as default in the workload cluster.
```
Comment (aarnq) [inline CHANGELOG.md:28]: This doesn't seem to be in this patch so it shouldn't be part of the changelog.

And there is no note of the fix for disabling `predictLinear`in the workload cluster.

Comment (Pavan-Gunda) [reply CHANGELOG.md]: I removed the backup operator change. and added that we disabled predict linear in workload cluster.

Comment (aarnq) [reply CHANGELOG.md]: I meant to add it under "Fixed", since it's now fixed so you can disable it there. We don't really disable it by default.

Comment (Pavan-Gunda) [reply CHANGELOG.md]: changed it.

**Review by aarnq — APPROVED** [review]

**Review by crssnd — APPROVED** [review]

**Pull request merged**